content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -0,0 +1,248 @@
+│ workspace               │ -            │      - │ 64               │
+│ cnn-48                  │ 09:11 AM     │ 0.9131 │ 48               │
+│ main                    │ Jul 21, 2021 │ 0.9189 │ 16               │
+│ 8583124                 │ Jul 20, 2021 │ 0.9132 │ 17               │
+└─────────────────────────┴──────────────┴────────┴──────────────────┘
+```
+
+### Keeping experiments in all tags
+
+When you tag the experiment commits by `git tag`, `dvc exp show --all-tags`
+presents them along with the tags. If you want the delete _all experiments
+without tags_, you can issue a `dvc exp gc --all-tags` command.
+
+```dvc
+$ dvc exp show --all-tags
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━━┓
+┃ Experiment              ┃    acc ┃ model.conv_units  ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━━┩
+│ workspace               │ 0.9067 │ 16                │
+│ ├── 2fc4f81 [exp-a1b3c4]│ 0.9037 │ 48                │
+│ └── 21beb69 [exp-d4e3ff]│ 0.9367 │ 128               │
+│ my-experiments          │ 0.9067 │ 16                │
+│ ├── 2fc4f81 [cnn-32]    │ 0.9067 │ 32                │
+│ ├── 5bc84a3 [cnn-64]    │ 0.9158 │ 64                │
+│ ├── 206cba6 [cnn-96]    │ 0.9260 │ 96                │
+│ └── 21beb69 [cnn-128]   │ 0.9379 │ 128               │
+└─────────────────────────┴────────┴───────────────────┘
+```
+
+```dvc
+$ dvc exp gc --all-tags
+$ dvc exp show --all-tags
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━━┓
+┃ Experiment              ┃    acc ┃ model.conv_units  ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━━┩
+│ workspace               │ 0.9067 │ 16                │
+│ my-experiments          │ 0.9067 │ 16                │
+│ ├── 2fc4f81 [cnn-32]    │ 0.9067 │ 32                │
+│ ├── 5bc84a3 [cnn-64]    │ 0.9158 │ 64                │
+│ ├── 206cba6 [cnn-96]    │ 0.9260 │ 96                │
+│ └── 21beb69 [cnn-128]   │ 0.9379 │ 128               │
+└─────────────────────────┴────────┴───────────────────┘
+```
+
+### Keeping experiments in all commits
+
+When you want to delete _all the experiments not associated with a Git commit_,
+you can do so by `--all-commits` flag. It deletes the experiments in the
+workspace that are not committed to the history.
+
+### Deleting Experiment-Related Objects in DVC Cache
+
+Note that `dvc exp gc` and `dvc exp remove` doesn't delete any objects in the
+DVC <abbr>cache</abbr>. In order to remove the cache objects, e.g. model files,
+intermediate artifacts, etc. related with the experiments, you can use `dvc gc`
+command.
+
+`dvc gc` receives the same _scoping_ flags, `--workspace`, `--all-branches`,
+etc. After a `dvc exp gc --workspace` command, you can supply
+`dvc gc --workspace` to remove all the experiment artifacts from the cache as
+well.
+
+## Removing Experiments in Remotes
+
+As you push the experiments with `dvc exp push`, remotes may be become cluttered
+with experiment references.
```

Comment:
We should link to https://dvc.org/doc/user-guide/experiment-management/sharing-experiments from somewhere areound here.